app/views/articles/edit.html.erb

```diff
@@ -28,6 +28,5 @@
 <% end %>
 
 <%= render "shared/webcomponents_loader_script" %>
-<%= javascript_pack_tag "clipboardCopy", defer: true %>
-<%= javascript_pack_tag "articleForm", defer: true %>
+<%= javascript_packs_with_chunks_tag "clipboardCopy", "articleForm", defer: true %>
```

Comment:
All packs on a page need to be included in the new helper. This prevents chunks from loading multiple times. For more information, see the [webpacker 4 upgrade guide](https://github.com/rails/webpacker/blob/master/docs/v4-upgrade.md).